app/templates/view_service.html

```diff
@@ -47,6 +47,20 @@
     {% endif %}
   {% endwith %}
 
+{% if service_data['status'] != 'published' and last_updated_by %}
+<div class="grid-row">
+  <div class="column-one-whole">
+    {%
+      with
+      type = "temporary-message",
+      heading = "Removed by {} on ".format(last_updated_by) + "{}.".format(last_updated_date|dateformat)|nbsp
```

Comment:
Is there a reason for not doing this as one string `"Removed by {} on {}."`?

Reply:
Yeah - the non breaking space on the date. If there is a less messy way to do it I am happy to change.